src/main/java/com/example/Homes/Web/UserDto.java

```diff
@@ -0,0 +1,63 @@
+package com.example.Homes.Web;
+
+public class UserDto {
+    private String userName;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+    public UserDto()
+    {
+
+    }
+
+    public UserDto(String userName, String firstName, String lastName, String email, String password) {
+        super();
```

Comment:
Is it necessary to call super class's constructor here? (it is not extending another class)